src/assets/styles/50-utilities/_links.scss

```diff
@@ -0,0 +1,11 @@
+// ----------------------------------
+// UI Settings
+// Links
+// ----------------------------------
+// Design System 1.0 Alpha
+// 2019-11-11
+// ----------------------------------
+
+.u-animated-link {
```

Comment:
I added this class to reuse it in CR (instead of copy-pasting the whole mixin)

Reply:
I think that makes sense given the use case. Applying `@include animated-link` explicitly on every anchor (e.g.`.cc-card__link`) or component which has anchor children (e.g. `.cc-rich-text a`) probably gives us better overall style control and of course there is less call for overriding it.

That said, if animated underline was applied globally instead (on `a { ... }` - in CR in this case) it would cascade down to every instance. Easier but potentially very annoying if overrides are ever needed.

You're closest to this, what do you think?

Reply:
@creido-welly 
Yes, if we apply underline globally in CR we will have to override a lot of links styles in the components. I read someone that if we have to undo a lot of styles it means there were added too soon (I was looking for that article, but can't find it now).

I left globally the `%anchor-base` styles on anchor in CR, in case I missed any link. (we are making sure that links have distinguish style even if missed out, and it should be easier to apply underline, once we spot them).

Reply:
Ok. We can review it in future and decide whether to keep as it is or change.

As an aside, SASS placeholders have been completely removed in design system. Where necessary they're being replaced with mixins.
https://github.com/wellcometrust/design-system/tree/develop/src/styles/20-tools